src/test/java/com/shapesecurity/shift/serialization/Serializer.java

```diff
@@ -179,7 +181,13 @@ JsonObjectBuilder add(@NotNull String property, @NotNull Either<StringBuilder, S
       return this;
     }
 
-    StringBuilder done() {
+    StringBuilder done(@NotNull Node node) {
+      SourceLocation loc = node.getLoc();
+      if (loc != null) {
+        this.add("range",
```

Comment:
It very well may be. If you have compelling use cases, open an issue on shift-spec.